mytoyota/models/endpoints/status.py

```diff
@@ -0,0 +1,49 @@
+""" Toyota Connected Services API - V1 Status Models """
+from datetime import datetime
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+from mytoyota.models.endpoints.common import StatusModel
+
+class _ValueModel(BaseModel):
+    value: str
+    status: int
+
+class _SectionModel(BaseModel):
+    section: str
+    values: List[_ValueModel]
+
+class _VehicleStatusModel(BaseModel):
+    category: str
+    display_order: int = Field(alias="displayOrder")
+    sections: List[_SectionModel]
+
+class _FugageModel(BaseModel):
```

Comment:
Have a look in common.py I've modelled this as its used in many places.

Reply:
thx for the hint. will have a look at it

Reply:
done: https://github.com/DurgNomis-drol/mytoyota/pull/242/commits/8f373417a6dea34b375f6c6075b12629f0d52129